pluto_build/sql/ownertype.sql

```diff
@@ -1,30 +1,9 @@
 -- set the owner type code based on data from COLP
-
--- set M owner tyoe
-UPDATE pluto a
-SET ownertype = 'M'
-FROM dcas_facilities_colp b
-WHERE b.code LIKE '%X%'
-AND a.bbl= b.bbl;
--- set P owner tyoe
-UPDATE pluto a
-SET ownertype = 'P'
-FROM dcas_facilities_colp b
-WHERE b.code LIKE '%P%'
-AND a.bbl= b.bbl;
--- set O as owner type
 UPDATE pluto a
-SET ownertype = 'O'
-FROM dcas_facilities_colp b
-WHERE b.code LIKE '%Q%'
-AND a.bbl= b.bbl;
--- set C as owner type
-UPDATE pluto a
-SET ownertype = 'C'
-FROM dcas_facilities_colp b
-WHERE b.type = 'OF'
-AND a.ownertype IS NULL
-AND a.bbl= b.bbl;
+SET ownertype = b.ownership
+FROM dcp_colp b
+WHERE a.bbl::numeric = b.bbl::numeric;
+
```

Comment:
The COLP ownership types don't map perfectly according to the [documentation](https://nycplanning.github.io/db-pluto/#/Data-Dictionary?id=ownertype), but do according to the latest version of `dcp_mappluto` in recipes. `O` versus `Other`.

Reply:
Would need to update documentation in this case